editor/src/components/shared/project-components.ts

```diff
@@ -391,19 +399,33 @@ export function getComponentGroups(
   // Add entries for dependencies of the project.
   for (const dependency of dependencies) {
     if (isResolvedNpmDependency(dependency)) {
-      const possibleComponentDescriptor = getThirdPartyComponents(
+      const dependencyStatus = getDependencyStatus(
+        packageStatus,
+        propertyControlsInfo,
         dependency.name,
-        dependency.version,
+        'loaded',
       )
-      if (possibleComponentDescriptor != null) {
-        const dependencyStatus = getDependencyStatus(
-          packageStatus,
-          propertyControlsInfo,
-          dependency.name,
-          'loaded',
-        )
-        const components =
-          dependencyStatus === 'loaded' ? possibleComponentDescriptor.components : []
+      const propertyControlsForDependency = propertyControlsInfo[dependency.name]
+      if (propertyControlsForDependency != null) {
+        const components = Object.keys(propertyControlsForDependency).map((name) => {
+          const elementNameAsArray = name.split('.')
+          const elementName =
+            elementNameAsArray.length > 0
+              ? jsxElementName(elementNameAsArray.shift()!, elementNameAsArray)
```

Comment:
Rather than doing this, please can we do:
```
const [baseVariable, ...propertyPathParts] = elementNameAsArray
...
jsxElementName(baseVariable, propertyPathParts)
```

Reply:
pushed a fix

Reply:
thanks, this is much nicer